contracts/ConsortiumSet.sol

```diff
@@ -55,6 +55,7 @@ contract ConsortiumSet is SystemValidatorSet, InitialConsortiumSet {
     uint internal constant maxValidators = 60;
     address[] private validatorsList;
     mapping(address => ValidatorStatus) private validatorsStatus;
+    address[] internal pendingList;
```

Comment:
I think we still gotta figure out how initialize the pending list, but perhaps thats just the next step.  Originally I was inheriting it because I wanted to template it out. If we pass it in from constructor is it going to be hard coded in committee set when we init? Hoping there is a simple way to do this for the chainspec.

Reply:
Ya so for testing, the committee set takes a list of initial consortium members. I think that makes sense as a design in plain English. When reading from the chainspec, that input parameter is ignored (can be empty) and instead pulls the deployed contract which will have the pendingList passed into the contract from the chainspec bytecode.